packages/utils/src/crypto/cryptoError.ts

```diff
@@ -0,0 +1,27 @@
+import { CRYPTO_DEBUG_MODE } from './constants'
+
+export class CryptoError extends Error {
```

Comment:
in my opinion this is not needed as discussed in tech channel

Reply:
changed, now obsolete